docs/semgrep-secrets/getting-started.md

```diff
@@ -1,153 +1,139 @@
 ---
 slug: /semgrep-secrets/getting-started
 append_help_link: true
-title: Scanning for secrets
+title: Scan for secrets
 hide_title: true
 description: Set up secrets scanning to find and rotate valid leaked secrets.
 tags:
   - Semgrep Secrets
-toc_max_heading_level: 2
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Getting started with Semgrep Secrets
+# Scan for secrets
 
-Detect and triage leaked secrets and credentials by scanning with Semgrep Secrets.
+Semgrep Secrets allows you to detect and triage leaked secrets and credentials
+and save time by prioritizing which secrets to rotate based on whether they're active and in use.
 
-Save time by prioritizing which secrets to rotate based on whether they are valid (active and in-use) or not.
+![Semgrep Secrets page](/img/secrets-page.png#md-width)
 
-![Semgrep Secrets page](/img/secrets-page.png)
-**_Figure._** Semgrep Secrets page. <br />
+This document guides you through:
 
-This document guides you through the following:
+1. Enabling Semgrep Secrets
+2. Viewing your results and triaging your findings
+3. Setting up PR comments and notifications
 
-* Enabling Semgrep Secrets
-* Receiving findings in GitHub PR comments
-* Triaging findings
+## Language and environment support
 
-To learn about how Semgrep Secrets detects secrets, see [<i class="fa-regular fa-file-lines"></i> Conceptual overview of Semgrep Secrets](/docs/semgrep-secrets/conceptual-overview).
+Semgrep Secrets can scan repositories using **any programming language** and supports the use GitHub, GitLab, and Bitbucket for PR and MR comments.
 
-:::info Feature maturity
-* This feature is in public beta.
-* You or your developers may encounter bugs. For issues, reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com).
-:::
+## Prerequisites
```

Comment:
One of the side effects of making the sidebars follow a series of steps is that the later sections no longer need the really long prerequisites they used to have. (Try comparing the articles in Notifications).

Do you think we could simplify this to either "You need to have finished a core deployment" or remove it ? 

Here's what the prereqs now look like in [other sections](http://semgrep.dev/docs/semgrep-code/semgrep-pro-engine-intro/):
 
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/f63f6be7-144d-4ad7-bc7b-d71399646c08)